### PR TITLE
Code clean up/pep8 clean up

### PR DESCRIPTION
**Pull request by ad-victoriam-101**
## Description

This Branch is just a small clean up of the DS code base to meet PEP8 standards on most .py files. 
Changes made were contained in the /app and  /data_generators folders. 
Added TODO calls for bug Fix branches in the future. 

[Loom Link](https://www.loom.com/share/01537f00de7b47d1b3bac56f2c4715e6)

## Type of change

- [x] Bug fix

## Checklist:

- [x] My code follows PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by tammer81 — APPROVED** [review]
This is the most beautifully-written and legible code I've ever seen; excellent job

**Review by cbell98 — APPROVED** [review]
Nice pep8 cleanup across files.

**Comment by ad-victoriam-101** [comment]
> This is the most beautifully-written and legible code I've ever seen; excellent job

Blessed be the PEP8 rules.

**Review by Lopez-John — APPROVED** [review]
Looks very clean. I like it. Good work

**Review by BobbyWilt — APPROVED** [review]

**Comment by amballa** [comment]
The code looks cleaner and more readable. Great job

**Comment by kevinbui100** [comment]
The code follows PEP8 guidelines and is definitely easier to read. Fantastic work!

**Review by kevinbui100 — APPROVED** [review]
The code follows PEP8 guidelines and is definitely easier to read. Fantastic work!

File: app/schema.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 from pydantic import BaseModel, constr, Extra
 
 
+# TODO add pydantic to requirements.txt
```
Comment (jinjahninjah) [inline app/schema.py:6]: Why not add Pydantic to the requirements.txt instead?

- go to the terminal in the IDE, make sure your virtual environment for the project is active
- enter the command `pip list`
- find pydantic
- I would enter it in the format of `pydantic~=<version number>`

File: data_generators/job_description_scraper.py [hunk 2]
```diff
@@ -18,6 +18,8 @@
 from webdriver_manager.chrome import ChromeDriverManager
 
 
+# TODO add selenium to requiremnts.txt or remove if not needed.
```
Comment (jinjahninjah) [inline data_generators/job_description_scraper.py:21]: Remove this as selenium is an avenue we cannot travel 😞 

There is an issue with selenium and the AWS service where AWS ElasticBeanstalk refreshing back to the original upload state. Reach out to Robert for more info 👍

File: data_generators/job_description_scraper.py [hunk 3]
```diff
@@ -524,7 +526,7 @@ def remove_job_titles(self, job_list):
                 job_list.append(parse.quote(job))
             self.search_terms = [
                 x for x in self.search_terms if x
-                not in job_list
+                                                not in job_list
```
Comment (jinjahninjah) [inline data_generators/job_description_scraper.py:529]: This change wasn't needed, revert and it should be good to go 👍

**Review by jinjahninjah — CHANGES_REQUESTED** [review]
Great job cleaning up some PEP8 issues from the codebase! There are few things to change:

- Instead of making a `TODO` about adding pydantic to the `requirements.txt`, go ahead and add it. Instructions in the in-line comment 👍 
- Remove the `TODO` about selenium as we can't use it with AWS. 
- There was a change made on line 529 of `data_generators/job_description_scraper.py` that needs to be reverted.  

Other than that it should be good to go 👍

**Comment by Ike888** [comment]
Nice clean code!

**Review by BrandonSmith710 — APPROVED** [review]
Nice job, it's looking very tidy.

File: app/schema.py (hunk 1 above)
Comment (ad-victoriam-101) [reply app/schema.py:6]: I can add it if needed. I wasn't sure if it was out of the scope of this branch to add those changes.

Comment (ad-victoriam-101) [reply app/schema.py:6]: I went a head an added it. It's on the most recent change.

File: data_generators/job_description_scraper.py (hunk 2 above)
Comment (ad-victoriam-101) [reply data_generators/job_description_scraper.py]: I removed the todo.

File: data_generators/job_description_scraper.py (hunk 3 above)
Comment (ad-victoriam-101) [reply data_generators/job_description_scraper.py]: Ah cats adding white space. It's been removed, fits inline with original indent.

**Review by jinjahninjah — APPROVED** [review]

**Pull request merged**